smoketests/helpers.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const path = require('path');
+
+const ROOT_PATH = process.env.GITHUB_WORKSPACE ? process.env.GITHUB_WORKSPACE : path.resolve(__dirname, '..');
+
+const getBinPath = () => path.resolve(ROOT_PATH, './packages/webpack-cli/bin/cli.js');
+
+const getPkgPath = (pkg) => {
+    const pkgPath = pkg.includes('webpack') ? `./node_modules/${pkg}` : `./node_modules/@webpack-cli/${pkg}`;
```

Comment:
We should use a separate boolean as argument (like `isSubPackage` which is `false` by default) instead of making decision based on whether it has webpack in name or not.